src/test/java/com/exasol/adapter/json/SchemaMetadataSerializerTest.java

```diff
@@ -1,21 +1,22 @@
 package com.exasol.adapter.json;
 
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.mockito.Mockito.when;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import javax.json.JsonObjectBuilder;
-
+import com.exasol.adapter.metadata.ColumnMetadata;
+import com.exasol.adapter.metadata.DataType;
+import com.exasol.adapter.metadata.SchemaMetadata;
+import com.exasol.adapter.metadata.TableMetadata;
```

Comment:
Use star import, more than four.

Reply:
For members we do this at three or more, but I would not do that for classes.